handlers/Contact/components/Locations/index.js

```diff
@@ -9,7 +9,7 @@ import locations from 'locationsData';
 var renderSegment: ReactElement = (segment: Location) => (
   <TabPanel key={segment.name}>
     <div className="ContactLocations-panel">
-      <img className="ContactLocations-map" src={`https://maps.googleapis.com/maps/api/staticmap?center=${segment.addr + ' ' + segment.location}&markers=label:Skookum%7C${segment.addr}&zoom=12&size=400x400`} />
+      <div className="ContactMap"><iframe src={segment.map}></iframe></div>
```

Comment:
It was requested we replace the images with interactive maps, so thats where that comes from. @shieldsma what do you think about linking static maps?